src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/Descriptor/alias_with_definition_2.txt

```diff
@@ -3,23 +3,24 @@
 [33mInformation for Service "[39m[32mservice_2[39m[33m"[39m
 [33m===================================[39m
 
- ------------------ --------------------------------- 
- [32m Option           [39m [32m Value                           [39m 
- ------------------ --------------------------------- 
-  Service ID         service_2                        
-  Class              Full\Qualified\Class2            
-  Tags               tag1 ([32mattr1[39m: val1, [32mattr2[39m: val2)  
-                     tag1 ([32mattr3[39m: val3)               
-                     tag2                             
-  Calls              setMailer                        
-  Public             no                               
-  Synthetic          yes                              
-  Lazy               no                               
-  Shared             yes                              
-  Abstract           no                               
-  Autowired          no                               
-  Autowiring Types   -                                
-  Required File      /path/to/file                    
-  Factory Service    factory.service                  
-  Factory Method     get                              
- ------------------ ---------------------------------
\ No newline at end of file
+ ------------------- ---------------------------------
+ [32m Option            [39m [32m Value                           [39m
+ ------------------- ---------------------------------
+  Service ID          service_2
+  Class               Full\Qualified\Class2
+  Tags                tag1 ([32mattr1[39m: val1, [32mattr2[39m: val2)
+                      tag1 ([32mattr3[39m: val3)
+                      tag2
+  Calls               setMailer
+  Public              no
+  Synthetic           yes
+  Lazy                no
+  Shared              yes
+  Abstract            no
+  Autowired           no
+  Autowired Methods   -
```

Comment:
Given the fact that the constructor is now always autowire, I think we need a single key, autowire (no "d"), that either contains true, or the list of methods, constructor present only if present in the user declaration.
If possible. WDYT?